### DISCUSSION TITLE
[Question] Force Fields Extensions Help

### DISCUSSION THREAD

**Discussion by shaoxiang** [isaac-sim/IsaacLab#1508] [Q&A]
Can the [Force Fields Extensions](https://docs.omniverse.nvidia.com/extensions/latest/ext_force-fields.html) be used in Isaac Lab? I tried to add Wind Force Field simulation in the Isaac Lab scene, but it didn't work. However, it works in Isaac Sim. I have enabled use_fabric. Are there any other configurations that need to be set? I would be very grateful for any useful help. Thanks.

**Comment by RandomOakForest** [comment]
Good question. Will move this into our _Discussions_ for follow up. Thanks for posting this.

**Comment by JunghwanRo** [comment]
Is there any update on this?
